src/Mollie/WC/Helper/IconFactory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Mollie\Api\Types\PaymentMethod;
+
+class Mollie_WC_Helper_IconFactory
+{
+
+    /**
+     * @param $data
+     * @return string
+     */
+    public function create($data)
```

Comment:
`$data` does not clarify what the variable contains, I would suggest to use a more explicit name for this variable.